tools/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -1805,9 +1805,26 @@ public void handlePath(String xpath) {
                 if (type == null) {
                     return; // no TTS, so keywords, skip
                 }
-                String name = cldrFileToFilter.getStringValue(xpath);
+                
                 XPathParts keywordParts = parts.cloneAsThawed().removeAttribute(2, "type");
                 String keywordPath = keywordParts.toString();
+                String rawKeywordValue = cldrFileToFilter.getStringValue(keywordPath);
+
+                // skip if keywords AND name are inherited
+                if (rawKeywordValue == null || rawKeywordValue.equals(CldrUtility.INHERITANCE_MARKER)) {
+                    String rawName = cldrFileToFilter.getStringValue(xpath);
+                    if (rawName == null || rawName.equals(CldrUtility.INHERITANCE_MARKER)) {
+                        return;
+                    }
+                }
+                
+                // skip if the name is not above root
+                String nameSourceLocale = resolved.getSourceLocaleID(xpath, null);
+                if ("root".equals(nameSourceLocale) || XMLSource.CODE_FALLBACK_ID.equals(nameSourceLocale)) {
+                    return;
+                }
```

Comment:
This is to prevent ↑↑↑ in individual emoji keywords, right? Wasn't that already fixed for the CLDR alpha3 last week?

Reply:
I thought it was, but it showed up in the PR. And we certainly want it in.

Reply:
I think this was because I mistakenly continued the branch. Will try to straighten out.

Reply:
Yeah, I was guessing that you had not rebased to master before starting the work for this PR, hopefully on a new branch